python/feathub/feature_tables/sinks/redis_sink.py

```diff
@@ -11,16 +11,19 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-from typing import Dict
+from typing import Dict, Union
 
+from feathub.common.exceptions import FeathubException
 from feathub.feature_tables.sinks.sink import Sink
+from feathub.feature_tables.sources.redis_source import RedisMode
 
 
 class RedisSink(Sink):
     def __init__(
         self,
         host: str,
         port: int = 6379,
+        mode: Union[RedisMode, str] = RedisMode.STANDALONE,
```

Comment:
Should we let user specify `name` for the table?

Reply:
Redis tends to use the notion `namespace` to identify keys[1], and this notion has been added to `RedisSink`'s constructor.

[1] https://redis.com/blog/5-key-takeaways-for-developing-with-redis/

Reply:
Sounds good.